03_Modules/Configuration/src/module/module.ts

```diff
@@ -61,29 +61,23 @@ import {
   IDeviceModelRepository,
   IMessageInfoRepository,
   sequelize,
+  SequelizeChargingStationSequenceRepository,
   ServerNetworkProfile,
   SetNetworkProfile,
 } from '@citrineos/data';
-import {
-  IdGenerator,
-  RabbitMqReceiver,
-  RabbitMqSender,
-  Timer,
-} from '@citrineos/util';
+import { IdGenerator, RabbitMqReceiver, RabbitMqSender } from '@citrineos/util';
 import { v4 as uuidv4 } from 'uuid';
-import deasyncPromise from 'deasync-promise';
 import { ILogObj, Logger } from 'tslog';
 import { DeviceModelService } from './DeviceModelService';
 import { BootNotificationService } from './BootNotificationService';
-import { SequelizeChargingStationSequenceRepository } from '@citrineos/data';
 
 /**
  * Component that handles Configuration related messages.
  */
 export class ConfigurationModule extends AbstractModule {
   public _deviceModelService: DeviceModelService;
 
-  protected _requests: CallAction[] = [
+  _requests: CallAction[] = [
```

Comment:
change for `_responses` is missing?

Reply:
🙊 not sure how I missed this, actually probably would work anyway, but it's good to stay consistent and not have unnecessary code / access modifier on the derived class as it will be inherited from the abstract base class - thank you catching this! 🙇

Reply:
> 🙊 not sure how I missed this, actually probably would work anyway, but it's good to stay consistent and not have unnecessary code / access modifier on the derived class as it will be inherited from the abstract base class - thank you catching this! 🙇

I see. 👍 Thank you for updating, Elliot!